test/unit/parserContext.test.ts

```diff
@@ -25,7 +25,8 @@ const cyclonedx_json = fs.readFileSync(
   'utf-8'
 );
 const cyclonedx_xml = fs.readFileSync(
-  path.join(getAppRootDir(), sbom_filepath, 'bom.cyclonedx.xml')
+  path.join(getAppRootDir(), sbom_filepath, 'bom.cyclonedx.xml'),
+  'utf-8'
```

Comment:
maybe this is just automatically generated, but what's the utf-8 tag here doing? are you decoding something in utf-8?

Reply:
yep! I'm reading sboms from sbom_examples folder. This call was missing the format specifier so I pushed the fix here.

more info on fs.readFileSync() :
https://www.geeksforgeeks.org/node-js-fs-readfilesync-method/